### PR TITLE
UOE-9680: Execute logger and append tracker pixel in creative for prebid.js DIY setup

### PR DESCRIPTION
**Pull request by AvinashKapre**

### CODE REVIEW

File: modules/pubmatic/openwrap/auctionresponsehook.go [hunk 1]
```diff
@@ -283,6 +280,20 @@ func (m OpenWrap) handleAuctionResponseHook(
 		result.DebugMessages = append(result.DebugMessages, string(rCtxBytes))
 	}
 
+	if rctx.Endpoint == models.EndpointWebS2S {
```
Comment (pm-isha-bharti) [inline modules/pubmatic/openwrap/auctionresponsehook.go:283]: Please add UTs for logger and tracker for webs2s endpoint

File: modules/pubmatic/openwrap/auctionresponsehook.go [hunk 2]
```diff
@@ -283,6 +280,20 @@ func (m OpenWrap) handleAuctionResponseHook(
...
+		result.ChangeSet.AddMutation(func(ap hookstage.AuctionResponsePayload) (hookstage.AuctionResponsePayload, error) {
+			rctx := moduleCtx.ModuleContext["rctx"].(models.RequestCtx)
+			var err error
+			ap.BidResponse, err = tracker.InjectTrackers(rctx, ap.BidResponse)
+			if err != nil {
+				return ap, err
+			}
+			resetBidIdtoOriginal(ap.BidResponse)
+			return ap, err
```
Comment (pm-jaydeep-mohite) [inline modules/pubmatic/openwrap/auctionresponsehook.go:292]: Can we restructure as  ?

`

    if err == nil {
 		resetBidIdtoOriginal(ap.BidResponse)
 	}		
 	return ap, err
 	
`

File: modules/pubmatic/openwrap/auctionresponsehook.go (hunk 1 above)
Comment (AvinashKapre) [reply modules/pubmatic/openwrap/auctionresponsehook.go:283]: Added test cases

File: modules/pubmatic/openwrap/auctionresponsehook.go (hunk 2 above)
Comment (AvinashKapre) [reply modules/pubmatic/openwrap/auctionresponsehook.go]: Yes, Done!

**Review by pm-isha-bharti — APPROVED** [review]

**Review by pm-saurabh-narkhede — APPROVED** [review]

**Pull request merged**